boardgame-ui/src/pages/WishListPage.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div>
+    <hero-title-page-layout>
+      <h1 class="title">{{$t('wish-list.title')}}</h1>
+      <h2 class="subtitle">{{$t('wish-list.subtitle')}}</h2>
+    </hero-title-page-layout>
+    <div class="container">
+      <b-loading :is-full-page="false" :active="loading"></b-loading>
+      <div class="section" v-if="!loading">
+        <board-game-list :board-games="boardGames" :allBelongToUser="true"
```

Comment:
Is there a reason why the "add from library" is disabled in the modal ?

Reply:
I thought it was more likely for the user to add board games not in his library to his wish list (because he wants other people to bring them to try them). But you're right, it might make sense to propose games from library. @hjwk, @FServais, what do you think?

Reply:
I definitely want to play board games that are in my library! :D Please please please, can we add it? :D

Reply:
Done, thanks! :-)